### PR TITLE
Release 0.9.0

### PR DESCRIPTION
**Pull request by damian5710**
### Added
- PNSSet class for parsing and writing `pnsSet.xml` files

### Fixed
- add ampSerialNumber and ampFirmwareVersion props to `FileInfo` class

### CODE REVIEW

**Review by GeorgeHernandez541 — DISMISSED** [review]

File: mffpy/xml_files.py [hunk 1]
```diff
@@ -1266,6 +1294,171 @@ def get_serializable_content(self):
...
+    <PNSSet xmlns="http://www.egi.com/pnsSet_mff"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+        <name>Physio 16 set 60hz 1.0</name>
+        <ampSeries>400</ampSeries>
+        <sensors>
+            <sensor>
+                <name>ECG</name>
+                <number>0</number>
+                <unit>uV</unit>
+                <psgType>0</psgType>
```
Comment (mscf) [inline mffpy/xml_files.py:1312]: Do you know what psgType is supposed to represent?

File: mffpy/xml_files.py [hunk 2]
```diff
@@ -1266,6 +1294,171 @@ def get_serializable_content(self):
...
+                <mapping>1</mapping>
```
Comment (mscf) [inline mffpy/xml_files.py:1313]: Does the number field represent the hardware channel number? What is mapping for?

**Review by mscf — APPROVED** [review]
I had a couple questions that I don't necessarily need answered. Otherwise, this looks good. Thanks!

**Review by GeorgeHernandez541 — APPROVED** [review]

File: mffpy/xml_files.py (hunk 1 above)
Comment (damian5710) [reply mffpy/xml_files.py:1312]: I'm not really sure. We should be able to define what this field means.

File: mffpy/xml_files.py (hunk 2 above)
Comment (damian5710) [reply mffpy/xml_files.py:1313]: If I'm not wrong, the number field indicates the channel index that should be used to get samples from the signal file.

**Pull request merged**